server/src/main/java/org/opensearch/repositories/blobstore/ChecksumBlobStoreFormat.java

```diff
@@ -221,7 +221,7 @@ public void writeAsync(
                     blobName,
                     bytes.length(),
                     true,
-                    WritePriority.HIGH,
+                    WritePriority.URGENT,
```

Comment:
For remote state lets pass the URGENT priority explicitly. Also add a comment that URGENT is used for remote state upload and should be used with proper justification.

Reply:
I have created 2 wrapper functions over this, one using this with urgent priority and other is using Normal priority. Added a Note with urgent method.